tests/acceptance/test_build.py

```diff
@@ -1235,3 +1235,56 @@ def test_mender_inventory_network_scripts(
         assert (
             b"mender-inventory-geo" not in output
         ), "mender-inventory-network-scripts unexpectedly a part of the image"
+
+    @pytest.mark.min_mender_version("2.7.0")
+    @pytest.mark.only_with_image("ext4")
+    def test_mender_dbus_interface_file(
+        self, request, prepared_test_build, bitbake_image
+    ):
+        """
+        Test the D-Bus interface file is provided by the mender-client-dev package,
+        but not installed by default.
+        """
+
```

Comment:
Same for the test, I would define here in the top a list of expected files (easy to locate, easy to add files) and the modify the asserts below for `any` and `all` kind of checks.